src/test/java/net/snowflake/ingest/streaming/internal/it/BigFilesITBase.java

```diff
@@ -17,16 +22,14 @@
 import net.snowflake.ingest.utils.Constants;
 import org.junit.After;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
 
 /** Ingest large amount of rows. */
 @RunWith(Parameterized.class)
-public class StreamingIngestBigFilesIT {
+public abstract class BigFilesITBase {
```

Comment:
is this passing?

Reply:
Yes it's passing with the JDBC uploader. The max chunk size in this test is 19MB.